src/resolution.rs

```diff
@@ -169,18 +169,36 @@ impl<CC : ChainComplex> ResolutionInner<CC> {
         if current_differential.next_degree() > t {
             // Already computed this degree.
             return
-        } else if current_differential.next_degree() > t {
-            // Haven't computed far enough yet
-            panic!(format!("We're not ready to compute bidegree ({}, {}) yet.", s, t));
-        }
+        } 
+        // else if current_differential.next_degree() > t {
+        //     // Haven't computed far enough yet

```

Comment:
why did you comment this out?

Reply:
It was causing me trouble at some stage in debugging. I see you put back in the match on compare statement with your force push.

I would have polished this more but my thesis is due today so I had to switch to that.